.github/workflows/test.yml

```diff
@@ -59,7 +60,7 @@ jobs:
           repository: qrtl/sbtechnology
           token: ${{ secrets.CI_PRIVATE_REPO_ACCESS_TOKEN }}
           path: sbtechnology
-          ref: refs/heads/master
+          ref: refs/heads/12.0-upd-license
```

Comment:
Is this branch correnct?

Reply:
I have to change if you have merged this branch.